operator/hack/common.sh

```diff
@@ -2,7 +2,7 @@
 # A library of bash functions useful for installing operator using OLM.
 
 declare -r KUTTL="${KUTTL:-kubectl-kuttl}"
-declare -r USE_MIDSTREAM_IMAGES=${USE_MIDSTREAM_IMAGES:-false}
+declare -r INSTALL_PLAN_APPROVAL=${INSTALL_PLAN_APPROVAL:-Manual}
```

Comment:
One more step is to pass `INSTALL_PLAN_APPROVAL` to the only function that needs it - to `apply_operator_manifests()`. Fewer globals simplify reading and understanding.

Reply:
The `main` function in `olm-operator-install.sh` also uses it.

Reply:
Yes, that's my point: it has to be lifted as explicit argument to `olm-operator-install.sh` as well.

Reply:
> it has to be lifted as explicit argument

I agree _it would be nice_ to eradicate env variables. But I don't think a "has to be lifted" is in the scope of this PR given it's not something I'm introducing here. Our CI-related scripts are full of env variable dependencies that I absolutely despise.  And I'm all for applying the boy-scout rule, but I think this is getting a bit too far.

Reply:
The variable is gone now.